docs/products/caching/concepts/overview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Aiven for Caching overview
+---
+
+Aiven for Caching, a fully managed **in-memory NoSQL database**, offers efficient storage and quick data access in your preferred cloud. Compatible with legacy Redis® OSS, this service facilitates seamless transitions and compatibility.
```

Comment:
and here

Reply:
Addded this. - This service is compatible with legacy Redis® OSS up to version 7.2.4, facilitating seamless transitions and compatibility.